skyportal/handlers/api/photometry.py

```diff
@@ -1062,16 +1063,37 @@ def delete(self, photometry_id):
 class ObjPhotometryHandler(BaseHandler):
     @auth_or_token
     def get(self, obj_id):
+        phase_fold_data = self.get_query_argument("phaseFoldData", False)
+
         Obj.get_if_accessible_by(obj_id, self.current_user, raise_if_none=True)
         photometry = Photometry.query_records_accessible_by(self.current_user).filter(
             Photometry.obj_id == obj_id
         )
         format = self.get_query_argument('format', 'mag')
         outsys = self.get_query_argument('magsys', 'ab')
+
         self.verify_and_commit()
-        return self.success(
-            data=[serialize(phot, outsys, format) for phot in photometry]
-        )
+        data = [serialize(phot, outsys, format) for phot in photometry]
+
+        if phase_fold_data:
+            period = None
+            annotations = (
+                Annotation.query_records_accessible_by(self.current_user)
+                .filter(Annotation.obj_id == obj_id)
+                .all()
+            )
+            for an in annotations:
+                if not isinstance(an.data, dict):
+                    continue
+                if 'period' in an.data:
```

Comment:
What about capital letters, e.g., `"Period"` as the key?

Reply:
modified, see what you think.